ably/realtime_conn_spec_test.go

```diff
@@ -100,14 +100,14 @@ func Test_RTN2_WebsocketQueryParams(t *testing.T) {
 		t.Parallel()
 		requestParams := setup()
 		libVersion := requestParams["v"]
-		assertDeepEquals(t, []string{"1.2"}, libVersion)
+		assertDeepEquals(t, []string{ably.AblyVersion}, libVersion)
 	})
 
 	t.Run("RTN2g: library and version should be included as the value of lib param", func(t *testing.T) {
 		t.Parallel()
 		requestParams := setup()
 		lib := requestParams["lib"]
-		assertDeepEquals(t, []string{"go-1.2.0"}, lib)
+		assertDeepEquals(t, []string{"go-" + ably.LibraryVersion}, lib)
```

Comment:
[As mentioned](https://github.com/ably/ably-go/pull/360#pullrequestreview-715264554), we shouldn't be sending the deprecated lib value anymore.

Reply:
Can you please mention which headers have been deprecated

Reply:
The whole [RSC7b](https://docs.ably.io/client-lib-development-guide/features/#RSC7b) spec item has been deprecated, so we should no longer be sending the "lib version" in either the `X-Ably-Lib` header or the `lib` query parameter.

Reply:
Done